server/src/server.ts

```diff
@@ -1392,8 +1403,15 @@ async function initPackageSearchPaths(uri: string) {
 
 		const settings = await getDocumentSettings(uri);
 		const sourceDir = settings.sourceDir;
+		const cairoPath = settings.cairoPath;
 
 		packageSearchPaths = '';
+
+		for (let i = 0; i < cairoPath.length; i++) {
+			packageSearchPaths += cairoPath[i];
```

Comment:
Looks like this should append `;` after each path as well.

Reply:
done